src/desktop/components/split_test/split_test.coffee

```diff
@@ -63,8 +60,6 @@ module.exports = class SplitTest
     outcome =
       if (@admin() and @edge?)
         @edge
-      else if @reflection()
-        if @control_group then @control_group else 'control'
```

Comment:
I actually like the idea of crawlers getting the "control" treatment of split tests, but it doesn't look like the code works that way. The `REFLECTION` value (below) is _only_ true for the specific `Artsy/Reflection` user-agent.

If there's a convenient way to identify crawlers here, we could keep this logic.